app/components/icons/YouTubeIcon.tsx

```diff
@@ -1,19 +1,18 @@
 export default function YouTubeIcon() {
   return (
     <svg
-      width="28"
-      height="20"
-      viewBox="0 0 28 20"
-      focusable="false"
+      width="164"
+      height="115"
+      viewBox="0 0 164 115"
       className="inline-block h-5 w-5 fill-current md:h-6 md:w-6"
+      focusable="false"
```

Comment:
Nit: `focusable` isn't really necessary, since by default the element isn't focusable

Not a big deal, I probably should do a quick a11y pass anyway to make sure these SVGs aren't causing any problems (such as trying to be read by a screenreader)